chapter06/docker-compose.yml

```diff
@@ -254,21 +229,19 @@ services:
         fi
         mkdir -p /sources/logs /sources/dags /sources/plugins
         chown -R "${AIRFLOW_UID}:0" /sources/{logs,dags,plugins}
-        chown -R "${AIRFLOW_UID}:0" /data
```

Comment:
I;d like to keep this so the chapters compose files are equal as much as possible

Reply:
indeed